src/webgpu/shader/validation/wgsl/variable-with-initilizer-storage.fail.wgsl

```diff
@@ -0,0 +1,12 @@
+# v-0032: variable 'u' has an initializer, however its storage class is 'storage'.
+
+[[block]]
+struct PositionBuffer {
+  [[offset(0)]] pos: vec2<f32>; 
+};
+
+var<storage_buffer> s : PositionBuffer = vec2<f32>(0.0, 0.0); 
```

Comment:
possible spec issue: I cannot find one use of "storage_buffer" in spec. Should it be "var<storage> s" instead.

Reply:
Correct, this was renamed to `storage` and tint hasn't caught up yet. (See https://github.com/gpuweb/gpuweb/pull/1093/files for the name update)